types/comtac/CM-3/lifecycle.schema.json

```diff
@@ -0,0 +1,76 @@
+{
+  "$id": "https://akenza.io/comtac/CM-3.lifecycle.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Comtac CM-3",
+  "type": "object",
+  "properties": {
+    "tempPt100": {
+      "title": "Temperature PT100",
+      "description": "TEMP PT100 placed",
+      "type": "boolean"
+    },
+    "tempI2C": {
+      "title": "Temperature I2C",
+      "description": "TEMP I2C placed",
+      "type": "boolean"
+    },
+    "acc": {
+      "title": "ACC",
+      "description": "ACC placed",
+      "type": "boolean"
+    },
+    "extMEM": {
+      "title": "EXT MEM",
+      "description": "EXT MEM placed",
+      "type": "boolean"
+    },
+    "lastTempValid": {
```

Comment:
this should not be in here -> move to default?

Reply:
These are only flags